### ISSUE TITLE
Material suffix not displayed in formly 6.0.0-next.4

### ISSUE THREAD

**Issue by mikejpeters** [ngx-formly/ngx-formly#3070]
**Description**
In Formly `6.0.0-next.4` the Material UI field suffix does not display. It was working in `6.0.0-next.1`.
This affects the `datepicker` type from `FormlyMatDatepickerModule` (as well as custom fields that set `to.suffix`):

![image](https://user-images.githubusercontent.com/3334654/144323167-bd1e51ce-62f0-4620-8b70-09ab197638fd.png)

**Minimal Reproduction**

```ts
// app.module.ts

@NgModule({
  imports: [
    BrowserModule,
    BrowserAnimationsModule,
    ReactiveFormsModule,
    MatNativeDateModule,
    FormlyMaterialModule,
    FormlyMatDatepickerModule,
    FormlyModule.forRoot(),
  ],
  declarations: [AppComponent],
  bootstrap: [AppComponent],
})
export class AppModule {}

// app.component.ts

@Component({
  selector: 'app-root',
  template: `<form [formGroup]="form">
    <formly-form [model]="model" [fields]="fields" [form]="form"></formly-form>
  </form>`,
})
export class AppComponent {
  form = new FormGroup({});
  model = {};

  fields: FormlyFieldConfig[] = [
    {
      key: 'date',
      type: 'datepicker',
      templateOptions: {
        label: 'Date',
      },
    },
  ];
}
```

**Your Environment**

- Angular 13.0.2
- @ngx-formly/core 6.0.0-next.4
- @ngx-formly/material 6.0.0-next.4

**Comment by mikejpeters** [comment]
Thanks for the quick fixes! Looking forward to the next release 🙂

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.0.0-next.5](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-next.5) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
